notebooks/ldb-eda-structure.ipynb

```diff
@@ -0,0 +1,1941 @@
+{
```

Comment:
<p>The bar plot gives us very valuable information, but I don't think so about the scatter plot. If your objective is to analyze the values distribution in the categories, maybe it would be better to see their standard deviation and their quartiles with the describe() method.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>